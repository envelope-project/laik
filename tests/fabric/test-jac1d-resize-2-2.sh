#!/bin/sh
timeout() { perl -e 'alarm shift; exec @ARGV' "$@"; }
timeout 5 ./launcher -n 2 -s 2 ../../examples/jac1d 100 50 -10 > test-jac1d-resize-2-2.out
cmp test-jac1d-resize-2-2.out "$(dirname -- "${0}")/test-jac1d-resize-2-2.expected"
