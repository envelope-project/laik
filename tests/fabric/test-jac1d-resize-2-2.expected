100 k cells (mem 1.6 MB), running 50 iterations with 2 tasks
Residuum after  1 iters: 299983.250000
Residuum after 11 iters: 195.693770
Residuum after 21 iters: 0.299484
Residuum after 31 iters: 0.056941
Residuum after 41 iters: 0.036303
Global value sum after 50 iterations: 299993.830104
